spec.bs

```diff
@@ -782,6 +782,21 @@ To <dfn>fill in a pending fenced frame config</dfn> given a [=fenced frame confi
 |pendingConfig|, [=auction config=] |auctionConfig|, [=leading bid info=] |winningBidInfo|, and
 [=auction report info=] |auctionReportInfo|:
 1. Let |winningBid| be |winningBidInfo|'s [=leading bid info/leading bid=].
+1. Let |replacements| be an [=ordered map=] whose [=map/keys=] are [=strings=] and whose 
+  [=map/values=] are [=strings=].
+1. [=list/For each=] |replacement| in |auctionConfig|'s 
+  [=auction config/deprecated render url replacements=]:
+  1. Let |k| be |replacement|'s [=ad keyword replacement/match=].
+  1. Let |v| be |replacement|'s [=ad keyword replacement/replacement=].
+  1. [=map/Set=] |replacements|[|k|] to |v|.
+1. Set |winningBid|'s [=generated bid/ad descriptor=]'s [=ad descriptor/url=] to the result of

```

Comment:
I think we cannot set |winningBid|'s [=generated bid/ad descriptor=], since I believe this still mutates |winningBidInfo|? Instead, create a new variable (e.g, |replacedAdUrl| or a better name), and replacing `|winningBid|'s [=generated bid/ad descriptor=]'s [=ad descriptor/url=]`  at current line 800 below with |replacedAdUrl|. Also replace ad descriptors with new variable |replacedAdComponentUrls| at current line 844 below to [=create nested configs=].